docs/understanding-tracking-design/managing-data-structures-with-data-structures-builder/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "🆕 Managing data structures with the Data Structures Builder"
+date: "2023-03-01"
+sidebar_position: 85
+---
+
+:::info
+The Data Structures Builder is only available for BDP Cloud Customers
+:::
+
+To create a new Data Structure, first navigate to **Data Structures** in the menu and click the **'Create a Data Structure'** button.
```

Comment:
This feels like it’s missing some link to other sections explaining what data structures are for (just imagining someone landing here with no context).

Reply:
this page sits in the Introduction to Tracking Design section, where they can find this into... theoretically they go through each section in order